bin/patch_tbprofiler_sv_output.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+
+import argparse
+import subprocess
+import pandas as pd
+import pysam
+import os
+import glob
+import json
+
```

Comment:
I have modularized @Wilma-chan script into different functions, easier to test and change.